src/Symfony/Component/Scheduler/Trigger/CallbackTrigger.php

```diff
@@ -20,11 +20,16 @@ final class CallbackTrigger implements TriggerInterface
 {
     private \Closure $callback;
 
-    public function __construct(callable $callback)
+    public function __construct(callable $callback, private string $description)
```

Comment:
I would not make it required. If not provided, let's use `spl_object_hash($tis->callback)`.